src/aleph/http.clj

```diff
@@ -482,7 +484,7 @@
    | `follow-redirects?`  | whether to follow redirects, defaults to `true`; see `aleph.http.client-middleware/handle-redirects`
    | `pool`               | a custom connection pool
    | `pool-timeout`       | timeout in milliseconds for the pool to generate a connection
-   | `connection-timeout` | timeout in milliseconds for the connection to become established
+   | `connection-timeout` | timeout in milliseconds for the connection to become established, defaults to 60s. Note that this timeout will be ineffective if the pool's `connect-timeout` is lower.
```

Comment:
Instead of only documenting, I would consider throwing an Exception if the `connection-timeout` is higher than the `connect-timeout`.

Reply:
I would have liked to do that but turns out that would require a deeper change: THe pool instance we have at this point doesn't expose the options that were passed to it for creation. I feel it's not worth the trouble for this alone. Or am I missing something perhaps?

Reply:
Fine for me!